Dashboard/Connectors/Serial/SerialConnector.cs

```diff
@@ -19,6 +24,12 @@ public void Start()
         // Set up the event handler for when data is received
         comPort.DataReceived += OnDataReceived;
 
+        // Setting up timer for heatbeat messages
```

Comment:
Do we need to be sending a heart beat every second, can we just check that the last received message was within some timeframe, and if not we send it?